packages/third-parties/update-third-parties.js

```diff
@@ -0,0 +1,108 @@
+const fs = require('fs/promises')
+const path = require('path')
+const AllThirdParties = require('third-party-capital')
+const prettier = require('prettier')
+
+const scriptStrategy = {
+  server: 'beforeInteractive',
+  client: 'afterInteractive',
+  idle: 'lazyOnload',
+  worker: 'worker',
+}
+
+const SRC = './src'
+const CONFIG_FILE_NAME = 'tpc-config.json'
+
+function generateComponent(thirdParty) {
+  let thirdPartyFunctions = ''
+
+  const insertScripts = (id, scripts, stylesheets) => {
+    let props = ''
+
+    if (stylesheets?.length > 0) {
+      props += ` stylesheets={${JSON.stringify(stylesheets)}}`
+    }
+
+    return scripts
+      .map((script) => {
+        if (typeof script === 'string') {
+          // External script with only URL
+          return `<Script src="${script}"${props} />`
+        } else if (script.url) {
+          // External script with additional properties
+          const { url, strategy } = script
+          return `<Script src={\`${url}\`} strategy="${scriptStrategy[strategy]}"${props} />`
```

Comment:
Maybe we can valid if the value of `scriptStrategy[strategy]` is legit before rendering? We can tackle this in the future, can just leave a TODO for now